imu_filter_madgwick/src/imu_filter.cpp

```diff
@@ -51,6 +51,10 @@ ImuFilter::ImuFilter(ros::NodeHandle nh, ros::NodeHandle nh_private):
   if (!nh_private_.getParam ("publish_debug_topics", publish_debug_topics_))
     publish_debug_topics_= false;
 
+  double orientation_stddev;
+  nh_private_.param<double>("orientation_stddev", orientation_stddev, 0.0);
+  orientation_variance_ = orientation_stddev * orientation_stddev;
+
```

Comment:
Yes, you can! Dynamic reconfigure reads the parameters from the parameter server on initialization.